src/index.js

```diff
@@ -485,7 +489,7 @@ const members = {
   },
   assignedSlot: {
     get () {
-      return nodeToSlotMap.get(this) || null;
+      return (nodeToSlotMap.get(this) && hostToModeMap.get(rootToHostMap.get(slotToRootMap.get(nodeToSlotMap.get(this)))) === 'open') ? nodeToSlotMap.get(this) : null;
```

Comment:
Can I suggest this be broken down into `const`s to help make this more readable?